wrappa/api_auth_wrappa.go

```diff
@@ -140,12 +142,17 @@ func (wrappa *APIAuthWrappa) Wrap(handlers rata.Handlers) rata.Handlers {
 			panic("you missed a spot")
 		}
 
-		if name == atc.GetAuthToken {
+		if name == atc.GetAuthToken || name == atc.GetAccessToken {
 			newHandler = auth.WrapHandler(newHandler, wrappa.getTokenValidator, wrappa.userContextReader)
 		} else {
 			newHandler = auth.WrapHandler(newHandler, wrappa.authValidator, wrappa.userContextReader)
 		}
-		wrapped[name] = auth.CSRFValidationHandler(newHandler, rejector, wrappa.userContextReader)
+		if name != atc.CheckResourceWebHook {
```

Comment:
So because of this line https://github.com/concourse/atc/blob/master/auth/cookie_set_handler.go#L16 it would only check that the `X-Csrf-Token` is specified. I have only tested locally in `no-auth` mode.